_config.yml

```diff
@@ -31,6 +31,9 @@ collections:
   publish:
     output: true
     permalink: /:collection/:name
+    subcategories:
+      - "Partner success stories"
+      UncategorizedPosition: "after"
```

Comment:
This block results in an error when I try running it locally -- could you provide doc links or a link elsewhere in this/another repo to this format? That way I can get a bit more context and help troubleshoot!